guide/invoicing/fees.mdx

```diff
@@ -72,3 +72,23 @@ This is illustrated below.:
   }
 }
 ```
+
+## Update fee payment status
+
+<Info>
+  The `payment_status` of a fee can only be configured if `pay_in_advance` is
+  `true` and `lago_invoice_id` is `null`.
```

Comment:
Technically, we don't enforce anything to `pay_in_advance`. It only check if the fee is attached to an invoice, `lago_invoice_id` must be null to update the fee.